libs/api/domains/auth-admin/src/lib/scope/models/scope-environment.model.ts

```diff
@@ -1,20 +1,64 @@
-import { Field, ObjectType } from '@nestjs/graphql'
+import { Field, ID, ObjectType } from '@nestjs/graphql'
 
 import { Environment } from '@island.is/shared/types'
-
 import { TranslatedValue } from '../../models/translated-value.model'
 
 @ObjectType('AuthAdminScopeEnvironment')
 export class ScopeEnvironment {
   @Field(() => Environment)
   environment!: Environment
 
-  @Field(() => String, { nullable: false })
+  @Field(() => Boolean)
+  enabled!: boolean
```

Comment:
Shouldn't the service just always return enabled scopes so we don't have to use this field?